File: pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);



    for (int i = 0; i < height; i++)
    {
        // adding the first leading space
        for (int j = height - i; j > 1 ; j--)
        {
            printf(" ");
        }

        // adding reverced peramid
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        // going to the new line
        printf("\n");
    }
}

File: pset6/sentimental/mario/less/mario.py
from cs50 import get_int

height = get_int("Height: ")

# check input
while height < 1 or height > 8:
    height = get_int("Height: ")


for i in range(height):
    # adding the first leading space
    print((" " * (height - i-1)) + ("#" * (i+1)))
